test/performance/odp_ipsecfwd.c

```diff
@@ -29,12 +30,12 @@
 #define MAX_SAS 64U
```

Comment:
Now that dest queue count is separately configurable, the max number of SAs could be increased significantly. Something like 4k would seem good to me.